src/main.py

```diff
@@ -1,2 +1,3 @@
 # python3.7.5
 print('hello world yashio!')
+
```

Comment:
**[reviewdog]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
W391 blank line at end of file